### PR TITLE
Update Grafana panels

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added or deleted? (Required)

 Update Grafana panels according to https://github.com/pingcap/dm/issues/543

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: zh/monitor-a-dm-cluster.md [hunk 1]
```diff
@@ -8,17 +8,13 @@ aliases: ['/docs-cn/tidb-data-migration/dev/monitor-a-dm-cluster/']
...
-> 目前只有 DM-worker 提供了 metrics，DM-master 暂未提供。
-
 ## Task
 
 在 Grafana dashboard 中，DM 默认名称为 `DM-task`。
 
 ### overview
 
-overview 下包含运行当前选定 task 的所有 DM-worker instance 的部分监控指标。当前默认告警规则只针对于单个 DM-worker instance。
+overview 下包含运行当前选定 task 的所有 DM-worker instance/source 的部分监控指标。当前默认告警规则只针对于单个 DM-worker instance/source。
```
Comment (csuzhangxc) [inline zh/monitor-a-dm-cluster.md:17]: 虽然我们把告警规则写在 `dm_worker.rules` 里，但也有包含 master 的了？

Comment (lance6716) [reply zh/monitor-a-dm-cluster.md:17]: 现在新家了 pending DDL 之类的告警，但是好像也不单属于 worker 或者 master……

File: zh/monitor-a-dm-cluster.md [hunk 2]
```diff
@@ -8,17 +8,13 @@ aliases: ['/docs-cn/tidb-data-migration/dev/monitor-a-dm-cluster/']
...
 
-> **注意：**
->
-> 目前只有 DM-worker 提供了 metrics，DM-master 暂未提供。
-
 ## Task
 
 在 Grafana dashboard 中，DM 默认名称为 `DM-task`。
 
 ### overview
```
Comment (ireneontheway) [inline zh/monitor-a-dm-cluster.md:15]:
```suggestion
### Overview
```

File: zh/monitor-a-dm-cluster.md [hunk 3]
```diff
@@ -30,28 +26,35 @@ overview 下包含运行当前选定 task 的所有 DM-worker instance 的部分
 | binlog file gap between master and syncer | 与上游 master 相比 binlog replication unit 落后的 binlog file 个数 | N/A | N/A |
 | shard lock resolving | 当前子任务是否正在等待 shard DDL 同步，大于 0 表示正在等待同步 | N/A | N/A |
 
-### task 状态
+### operate error
```
Comment (ireneontheway) [inline zh/monitor-a-dm-cluster.md:29]:
```suggestion
### Operate error
```

File: zh/monitor-a-dm-cluster.md [hunk 4]
```diff
@@ -30,28 +26,35 @@ overview 下包含运行当前选定 task 的所有 DM-worker instance 的部分
...
-| write relay log duration | relay log 每次写 binlog 到磁盘的时延，单位：秒| N/A | N/A |
-| binlog event size | relay log 写到磁盘的单条 binlog 的大小 | N/A | N/A |
+| number of dm-masters start leader components per minute | 每分钟内 DM-master 尝试启用 leader 相关组件次数 | N/A | N/A |
+| number of workers in different state | 不同状态下有多少个 DM-worker | 存在离线的 DM-worker 超过一小时 | critical |
+| workers' state | DM-worker 的状态 | N/A | N/A |
+| number of worker event error | 不同类型的 DM-worker 错误出现次数 | N/A | N/A |
+| shard ddl error per minute | 每分钟内不同类型的 shard DDL 错误次数 | 发生 shard DDL 错误 | critical |
+| number of pending shard ddl | 未完成的 shard DDL 数目 | 存在未完成的 shard DDL 数目超过一小时 | critical |
+
+### task 状态
```
Comment (ireneontheway) [inline zh/monitor-a-dm-cluster.md:53]:
```suggestion
### Task 状态
```

**Pull request merged**